byway/src/main/webapp/destinations.js

```diff
@@ -32,6 +32,9 @@ function initializeDestinationsPage(){
     updateStartDestination(response);
   }); 
 }
+
+const urlParams = new URLSearchParams(window.location.search);
+const tripKey = urlParams.get('tripKey');
```

Comment:
Other pages seem to be using `tripId` rather than `tripKey`, please discuss on team and be consistent.

Reply:
we discussed a few weeks back and had decided that tripKey makes more sense bc we are using the Key from datastore, but I will double check that everyone is ont he same page